### ISSUE TITLE
S1128 False Positive when MAML references code

### ISSUE THREAD

**Issue by jcurl** [SonarSource/sonar-dotnet#2759]
### Description

S1128 is shown as a warning, even if MAML might required the using statement.

### Repro steps

```csharp
using System.Collections.Generic;

/// <summary>
/// A linked list node for our block list.
/// </summary>
/// <remarks>
/// This class is not 100% compatible to <seealso cref="LinkedListNode{T}"/>.
/// </remarks>
public class MyClass {
  ...
}
```

### Expected behavior

The recommendation to remove the warning should be suppressed.

### Actual behavior

The statement `using System.Collections.Generic` is marked as unused with S1128, even though it is used in the MAML comments. Removing the using will result in CS1574 compiler warning.

### Known workarounds

None.

### Related information

* SonarC# Version 4.13.0.11687
* Visual Studio Version 2019 version 16.3.6
* If running through the Scanner for MSBuild, its version N/A

**Comment by andrei-epure-sonarsource** [comment]
thanks @jcurl , this is a duplicate of #2694 ; closing as duplicate
